Machine_Learning/Logistic_Regression/Logistic Regression.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
Make the following changes:
1. use `np.concatenate()` function instead of creating an array of zeros for `dist_01` and `dist_02`
2. in `get_grad()` function you are returning a python list. Either change your comment or return type.
3. in the `gradient_descent()` function you should subtract the gradients from weights. Change sign to minus.
4. change function name `get_acc()` to `accuracy()`.
5. in loss curve, x label should be `epoch`. Also, reduce the number of epochs. You are getting a saturated curve after 200 epochs.
6. add the axis label in the accuracy curve.